app/components/gleam-logo.hbs

```diff
@@ -0,0 +1,9 @@
+<div
+  class="gleam-logo-container"
+  {{did-insert this.setupRive}}
```

Comment:
Our convention for handlers is to always name them after the event (i.e. something like  `this.handleDidInsert` in this case). Let's us now have to worry about naming each of these + allows us to easily edit handlers to add more functionality without having to rename them.

Reply:
noted!